src/_playground/documentation/Import/Import.js

```diff
@@ -1,21 +1,24 @@
+import getSourceModules from '../utils/getSourceModules';
 import { googlecode } from 'react-syntax-highlighter/styles/hljs';
 import packageJson from '../../../../package.json';
 import PropTypes from 'prop-types';
 import React from 'react';
 import SyntaxHighlighter from 'react-syntax-highlighter';
 
 export const Import = ({ sourceModule }) => {
+    // remove everything up until the last forward slash and the file extension.
+    const importPath = sourceModule.replace(/^(.*[\\\/])/, '').replace('.js', '');
 
     return (
         <SyntaxHighlighter
             customStyle={{ padding: 0 }}
             language='javascript'
             style={googlecode}>
-            {`import { ${Object.keys(sourceModule).sort().join(', ')} } from '${packageJson.name}';`}
+            {`import { ${Object.keys(getSourceModules(sourceModule)).sort().join(', ')} } from '${packageJson.name}/lib/${importPath}';`}
         </SyntaxHighlighter>
     );
 };
 
 Import.propTypes = {
-    sourceModule: PropTypes.object.isRequired
+    sourceModule: PropTypes.string.isRequired
```

Comment:
Since this is being passed in as a string now, the prop name should probably change as well.  Similar to the new `getSourceModules` util function, maybe use `sourceModulePath`?